hyperspy/Release.py

```diff
@@ -25,7 +25,7 @@
 # When running setup.py the ".dev" string will be replaced (if possible)
 # by the output of "git describe" if git is available or the git
 # hash if .git is present.
-version = "1.7.2.dev0"
+version = "1.8.0.dev0"
```

Comment:
RELEASE_next_patch should stay on 1.7.2.dev0

This and the next change look like you had started out from RELEASE_next_minor, which would explain the errors you got when trying to PR into RELEASE_next_patch

Reply:
Hi, yes, the only branch I could fork was the RELEASE_next_minor, the other were not listed, how to start out from the the RELEASE_next_patch instead? *first time on Github

Reply:
There are different ways of doing it, but here are the main steps:

1. Checkout the branch off `RELEASE_next_patch`, for example: `git checkout -b my_new_branch RELEASE_next_patch`
2. commit your changes
3. push your changes to your own repository (should be `https://github.com/OliDG/hyperspy`)
4. on github, make the pull request to `https://github.com/hyperspy/hyperspy` from your branch in `https://github.com/OliDG/hyperspy`

Usually, the term fork is used in the context of repository, not branch.

Reply:
If you have issue with this, for this time, you can continue to commit and push to this branch and we can "clean" the branch before merging, once we are happy with the content of this PR.